### PR TITLE
chore: remove references to "experimental" analysis

### PR DESCRIPTION
**Pull request by ivanstanev**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Remove references to "experimental" in code since we don't have such a code path any more.

### CODE REVIEW

**Review by agatakrajewska — APPROVED** [review]

File: lib/image-save-path.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as path from "path";
```
Comment (agatakrajewska) [inline lib/image-save-path.ts:1]: 👍

**Review by shaimendel — APPROVED** [review]

**Review by dkontorovskyy — APPROVED** [review]

File: test/system/static.test.ts [hunk 2]
```diff
@@ -359,55 +357,6 @@ test("static analysis for distroless base-debian10", async (t) => {
   );
 });
 
-test("experimental static analysis for debian images", async (t) => {
```
Comment (agranado2k) [inline test/system/static.test.ts:362]: I'm wondering why we don't need this test anymore. 🤔

Comment (ivanstanev) [reply test/system/static.test.ts:362]: Good spot! For two reasons:
- we don't have "experimental" static analysis flow, we have only 1 flow and that is "static analysis"
- we already have tests that check the default static analysis flow for both image identifier and image SHA (and archives)

**Review by agranado2k — DISMISSED** [review]
Great job! LGTM! 🎉

**Review by agranado2k — APPROVED** [review]
LGTM! 🎉

**Pull request merged**